runtime/doc/news.txt

```diff
@@ -65,6 +65,10 @@ The following changes may require adaptations in user config or plugins.
   now requires an explicit range argument to be passed. If injections are
   required, provide an explicit range via `parser:parse({ start_row, end_row })`.
 
+• The minimum version of glibc required to run the linux releases has been
```

Comment:
This is specific to our prebuilt release workflow, not neovim itself.  The wording should probably make that clear. On the other hand, since this is specific to the binaries we build, maybe it should just go in the GH release notes instead of the shipped release notes.